rgbd/tools/rgbd_to_pcd.cpp

```diff
@@ -0,0 +1,84 @@
+#include <rgbd/serialization.h>
+#include <rgbd/image.h>
+#include <rgbd/view.h>
+
+#include <fstream>
+#include <iostream>
+
+#include <pcl/io/pcd_io.h>
+#include <pcl/point_types.h>
+
+#include <cmath>
+
+#include <opencv2/highgui/highgui.hpp>
+
+#include <typeinfo>
+
+
+int main(int argc, char **argv) {
+
+    if (argc < 2)
+    {
+        std::cout << "Usage:\n\n   rgbd_to_pcd FILENAME.rgbd (optional)SECOND_FILENAME.rgbd\n\n";
```

Comment:
```suggestion
        std::cout << "Usage:\n\n   rgbd_to_pcd FILENAME.rgbd [FILENAME2.rgbd ...]\n\n";
```
This is a more common way to represent the optional extra arguments

Reply:
Thanks